llvm/lib/IR/Function.cpp

```diff
@@ -1482,8 +1482,8 @@ static Type *DecodeFixedType(ArrayRef<Intrinsic::IITDescriptor> &Infos,
     return VectorType::getSubdividedVectorType(VTy, SubDivs);
```

Comment:
Are the changes in the file formatting only? If so, Can this be removed from this patch?

Reply:
Sure, we'll see if the format checks complain.